src/translators/sql_translator.py

```diff
@@ -543,12 +549,13 @@ def _get_where_clause(self, entity_ids, from_date, to_date, fiware_sp=None,
         if geo_clause:
             clauses.append(geo_clause)
 
-        where_clause = "where " + "and ".join(clauses)
+        if len(clauses) > 0:
+            where_clause = "where" + " and ".join(clauses)
         return where_clause
 
     def _parse_date(self, date):
         try:
-            return dateutil.parser.isoparse(date.strip('\"'))
+            return dateutil.parser.isoparse(date.strip('\"')).isoformat()
```

Comment:
in this way the we get the correct iso format compatible with both crate 3.x and crate 4.x

Reply:
great, I'll have to see what happens with Timescale. I'll wait until we merge this PR.